cadastre/cadastre_import.py

```diff
@@ -1009,7 +1009,25 @@ def unzipFolderContent(self, path):
                 for z in tarFileListA:
                     with tarfile.open(z) as t:
                         try:
-                            t.extractall(os.path.join(self.edigeoPlainDir, 'tar_%s' % i))
+                            arguments = {}
+                            if sys.version_info < (3, 9, 17) or sys.version_info < (3, 10, 12):
```

Comment:
The test is always true if the version is < (3, 10, 12) whatever the minor version of python and thus a warning is issued even if the python version is ok (as python 3.9.18).